.github/workflows/e2e_testnet_daily.yml

```diff
@@ -141,16 +142,17 @@ jobs:
         get_solana_sha,
       ]
     container:
```

Comment:
should this be commented out too?

Reply:
Good catch - commented them all out. 9c551f9